hera_sim/visibilities/vis_cpu.py

```diff
@@ -34,8 +33,20 @@ def __init__(self, bm_pix=100, real_dtype=np.float32,
             Arguments of :class:`VisibilitySimulator`.
         """
 
-        self._real_dtype = real_dtype
-        self._complex_dtype = complex_dtype
+        assert precision in (1,2)
+        self._precision = precision
+        if precision == 1:
+            self._real_dtype = np.float32
+            self._complex_dtype = np.complex64
+        else:
+            self._real_dtype = np.float64
+            self._complex_dtype = np.complex128
+
+        if use_gpu:
+            from hera_gpu.vis import vis_gpu
```

Comment:
Given that I would expect `hera_gpu` to be an optional depency, we should wrap this import with a `try: except ImportError:` and raise with some instruction on how to install GPU support (I expect it will be something like `pip install .[gpu]`.

Reply:
Added a generic statement. Feel free to tweak if desired.